code/game/objects/items/knives.dm

```diff
@@ -279,3 +292,18 @@
 /obj/item/knife/shiv/carrot/suicide_act(mob/living/carbon/user)
 	user.visible_message(span_suicide("[user] forcefully drives \the [src] into [user.p_their()] eye! It looks like [user.p_theyre()] trying to commit suicide!"))
 	return BRUTELOSS
+
+/obj/item/knife/shiv/parsnip
+	name = "parsnip shiv"
+	icon_state = "parsnipshiv"
+	inhand_icon_state = "parsnipshiv"
+	desc = "Truly putting 'snip' in the 'parsnip', and it's not sub-par either!"
+	custom_materials = null
+
+/obj/item/knife/shiv/root
+	name = "cahn'root shiv"
+	icon_state = "rootshiv"
+	inhand_icon_state = "rootshiv"
+	desc = "A root sharpened into a shiv. A root source of someone's stab wounds soon, most likely."
+	custom_materials = null
```

Comment:
Why have two of these if they are identical except for visuals? I get the idea of adding cool stuff but I worry about adding too much here

Reply:
Three of these with the carrot; consistancies sake. I wondered if I could diversify them, but decided not to complicate it.

Reply:
alllright